src/lib/y2network/interface_config_builders/bond.rb

```diff
@@ -19,10 +19,22 @@ def bondable_interfaces
         interfaces.all.select { |i| bondable?(i) }
       end
 
+      attr_writer :bond_options
+      def bond_options
+        return @bond_options if @bond_options
+
+        @bond_options = Yast::LanItems.bond_option
+        if @bond_options.nil? || @bond_options.empty?
+          @bond_options = @config["BONDING_MODULE_OPTS"]
+        end
+        @bond_options
+      end
+
       def save
         super
 
         Yast::LanItems.bond_slaves = @config["BOND_SLAVES"]
+        Yast::LanItems.bond_option = bond_options
```

Comment:
It looks tricky to me that one reads from @config and the other depends on if it is in LanItems or not..

Reply:
in the end they are same, both are read value or modified one. I just do not create shortcut for bond_slaves for now.